packages/ckeditor5-bookmark/tests/editbookmarkcommand.js

```diff
@@ -0,0 +1,110 @@
+/**
+ * @license Copyright (c) 2003-2024, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/* global document */
+
+import { Essentials } from '@ckeditor/ckeditor5-essentials';
+import { Paragraph } from '@ckeditor/ckeditor5-paragraph';
+import { Heading } from '@ckeditor/ckeditor5-heading';
+import { ClassicEditor } from '@ckeditor/ckeditor5-editor-classic';
+import { getData as getModelData, setData as setModelData } from '@ckeditor/ckeditor5-engine/src/dev-utils/model.js';
+import { Bold, Italic } from '@ckeditor/ckeditor5-basic-styles';
+
+import BookmarkEditing from '../src/bookmarkediting.js';
+import EditBookmarkCommand from '../src/editbookmarkcommand.js';
+
+describe( 'EditBookmarkCommand', () => {
+	let domElement, editor, model, command;
+
+	beforeEach( async () => {
+		domElement = document.createElement( 'div' );
+		document.body.appendChild( domElement );
+
+		editor = await ClassicEditor.create( domElement, {
+			plugins: [
+				Paragraph,
+				Heading,
+				Essentials,
+				BookmarkEditing,
+				Bold,
+				Italic
+			]
+		} );
+
+		model = editor.model;
+		command = new EditBookmarkCommand( editor );
+	} );
+
+	afterEach( () => {
+		domElement.remove();
+		return editor.destroy();
+	} );
+
+	describe( '#isEnabled()', () => {
+		describe( 'should be `true`', () => {
+			it( 'when only bookmark element is selected', () => {
+				setModelData( model, '<paragraph>[<bookmark bookmarkId="foo"></bookmark>]</paragraph>' );
+
+				expect( command.isEnabled ).to.equal( true );
+			} );
+		} );
+
+		describe( 'should be `false`', () => {
```

Comment:
Maybe some additional tests of some other object selected could be useful here.

Reply:
Added `Link` and `imageInline`.